openedx/features/partners/forms.py

```diff
@@ -0,0 +1,23 @@
+from django import forms
+from django.contrib.auth.models import User
+
+
+class PartnerResetPasswordForm(forms.Form):
```

Comment:
@abdullah998 instead is there any specific reason write a new form for G2A, if not please look into PasswordResetFormNoActive it's already being used in our reset email. To customize messages you can override this form here.

Reply:
G2A has a unique requirement where we have to tell the user if the user against that email does not exists in our database instead of the generic error "If this email exists in our database we have sent an email". Because of that validation, I have to use this form. I could just do it in the view, but the way front-end validations are implemented requires the error in a specific format, one that this form gives. @emraan004

Reply:
Done.